solr/server/etc/jetty-https.xml

```diff
@@ -66,8 +66,8 @@
         <Set name="host"><Property name="solr.host.bind" default="127.0.0.1"/></Set>
         <Set name="port"><Property name="solr.jetty.https.port" default="8983" /></Set>
         <Set name="idleTimeout"><Property name="solr.jetty.https.timeout" default="120000"/></Set>
```

Comment:
Note that this timeout property is different than the new one you are adding, for the same thing